mealie/core/settings/settings.py

```diff
@@ -68,23 +68,23 @@ def DB_URL_PUBLIC(self) -> str | None:
     SMTP_HOST: Optional[str]
     SMTP_PORT: Optional[str] = "587"
     SMTP_FROM_NAME: Optional[str] = "Mealie"
-    SMTP_TLS: Optional[bool] = True
+    SMTP_TLS: Optional[bool] = False
     SMTP_FROM_EMAIL: Optional[str]
     SMTP_USER: Optional[str]
     SMTP_PASSWORD: Optional[str]
+    SMTP_AUTH_STRATEGY: Optional[str] = "TLS"  # Options: 'TLS', 'SSL', 'NONE'
+
+    if SMTP_TLS:
+        SMTP_AUTH_STRATEGY = "TLS"
```

Comment:
I'm wondering if we even need to do this? 

Today, we default to use TLS, so I am defaulting the new Auth to TLS. No change in this case. 
Today, if someone set `SMTP_TLS` to False, then should I be setting Auth Strategy to none?

Reply:
I'm also wondering if we need to do a warning console statement if SMTP_TLS is used? What is the preferred deprecation strategy?

Reply:
I think at this point we could probably just remove the SMTP_TLS and just put a deprecation notice in the change-log. From your description it sounds like it shouldn't be too much of an issue, even if folks were using it.

Reply:
That's what I'll do then to just avoid creating tech debt.